tests/transformers/helper.py

```diff
@@ -118,6 +118,9 @@ def load_component_multi_modal():
 @prefetch
 @flaky()
 def load_small_conversational_model():
+    # Conversational model is deprecated and removed
+    if Version(transformers.__version__) > Version("4.41.2"):
```

Comment:
```suggestion
    if _try_import_conversational_pipeline() is None:
```

can we use `_try_import_conversational_pipeline`?